libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineApi.kt

```diff
@@ -872,7 +882,7 @@ class MapboxRouteLineApi(
     private suspend fun buildDrawRoutesState(
         featureDataProvider: () -> List<RouteFeatureData>
     ): Expected<RouteLineError, RouteSetValue> {
-        val routeFeatureDataDef = jobControl.scope.async(ThreadController.IODispatcher) {
+        val routeFeatureDataDef = jobControl.scope.async {
```

Comment:
Are we ok using `Default` dispatcher here instead of `IO`?

Same question applies to other occurrences below 👇

Reply:
IMO, since there's no IO here there's no reason to use an IO dispatcher.  The default JVM thread pool should be sufficient for these tasks I don't see any reason to use the larger IO thread pool.

Reply:
Cool that makes sense. Thanks for clarifying 🙇‍♂️ 

Yeah, also `Default` and `IO` share resources 🚀